index.html

```diff
@@ -2,10 +2,200 @@
+        <img
+          src="./images/Profile1.jpg"
+          height="600px"
+          width="600px"
+          alt="My avatar"
+        />
+      </figure>
+      <aside class="intro">
+        <span id="intro1"> </span><br />
+        <span id="intro2"> </span><br />
+        <span id="intro3"> </span>
+      </aside>
+    </section>
+
+    <section class="icons">
+      <ul>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/madhuri-m-96b89481/"
+            target="blank"
+          >
+            <img src="./images/icons8-linkedin-48.png"
+          /></a>
+        </li>
+        <li>
+          <a href="https://github.com/madhurimt" target="blank">
+            <img src="./images/icons8-github-48.png"
+          /></a>
+        </li>
+        <li>
+          <a href="mailto:madhuri.maniktahla@gmail.com?" target="blank">
+            <img src="./images/icons8-mail-48.png"
+          /></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/icons8-home-48.png" /></a>
+        </li>
+      </ul>
+    </section>
+
+    <section class="content">
+      <section id="about" class="pagestyle">
+        <h2>About</h2>
+        <p>
+          Hi,I am Madhuri Maniktahla. I am a front-end web developer with
+          expertise in JavaScript, HTML, and CSS. I have recently completed a
+          coding bootcamp and have a degree in computer science engineering. I
+          also have experience as an application support engineer and possess
+          project and time management skills. I spend my time designing and
+          coding web applications and enjoy outdoor activities in my free time.
+        </p>
+      </section>
+
+      <section id="work" class="pagestyle">
+        <h2>Work</h2>
+
+        <figure class="projects">
+          <div class="card">
+            <ul>
+              <li class="card1">
+                <h3>Weather Dashboard</h3>
+                <p>
+                  A simple weather dashboard to display current weather of
+                  different cities<br /><br />
+                  HTML, CSS, JavaScript, Open Weather API
+                </p>
+                <img src="./images/weather.jpg" alt="Weather App" />
+                <h6>
+                  <a
+                    href="https://weather-project-xi.vercel.app/"
+                    target="blank"
+                    >Application</a
+                  >
+                  ||
+                  <a
+                    href="https://github.com/madhurimt/weather_project"
+                    target="blank"
+                    >Repository
+                  </a>
+                </h6>
+              </li>
+              <li class="card1">
+                <h3>Breathing App</h3>
+                <p>
+                  An application for users to meditate and relax.<br /><br />
+                  HTML, CSS, JavaScript
+                </p>
+                <img src="./images/meditation.jpg" alt="Meditation App" />
```

Comment:
I suggest using some css to fix the stretched images used for your portfolio images